devimint/src/main.rs

```diff
@@ -592,6 +592,52 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
     Ok(())
 }
 
+async fn lightning_gw_reconnect_test(dev_fed: DevFed, process_mgr: &ProcessManager) -> Result<()> {
+    #[allow(unused_variables)]
+    let DevFed {
+        bitcoind,
+        cln,
+        lnd,
+        fed,
+        gw_cln,
+        gw_lnd,
+        electrs,
+        esplora,
+    } = dev_fed;
+
+    drop(cln);
+    drop(lnd);
+
+    let gateways = vec![gw_cln, gw_lnd];
+
+    for mut gw in gateways {
+        // Verify that the gateway can query the lightning node for the pubkey and alias
+        let mut cmd = gw.cmd().await.arg("info");
+        assert_eq!(cmd.run().await.is_ok(), true);
+
+        // Verify that after stopping the lightning node, info no longer returns since
+        // the lightning node is unreachable.
+        gw.stop_lightning_node().await?;
+        let info_fut = cmd.run();
```

Comment:
interesting reuse of cmd, confused me a lot.

Reply:
Changed to your suggestion below